webtau-config/src/main/java/com/twosigma/webtau/cfg/WebTauConfig.java

```diff
@@ -46,6 +46,11 @@
     private final ConfigValue waitTimeout = declare("waitTimeout", "wait timeout in milliseconds", () -> 5000);
     private final ConfigValue disableFollowingRedirects = declareBoolean("disableRedirects", "disable following of redirects from HTTP calls");
     private final ConfigValue maxRedirects = declare("maxRedirects", "Maximum number of redirects to follow for an HTTP call", () -> 20);
+    private final ConfigValue userAgent = declare("userAgent", "User agent to send on HTTP requests",
+            () -> "webtau/" + getClass().getPackage().getImplementationVersion());
+    private final ConfigValue removeWebtauFromUserAgent = declare("removeWebtauFromUserAgent",
```

Comment:
was wondering why enable is a toggle and disable is default. It is a bit unusual for REST api to expect user agent.

Reply:
I agree that it's unlikely an API will expect an agent but it's useful for identifying requests from tests in logs.